OpenRA.Game/Widgets/Widget.cs

```diff
@@ -162,6 +173,10 @@ public abstract class Widget
 		public bool Visible = true;
 		public bool IgnoreMouseOver;
 		public bool IgnoreChildMouseOver;
+		public Color DebugBoundsColor = Color.Red;
+		public float DebugBoundsWidth = 1;
+		public Color DebugMouseOverBoundsColor = Color.Green;
+		public float DebugMouseOverBoundsWidth = 2;
```

Comment:
What's the logic behind having this per-widget?  This makes it a real pain to adjust mod-wide.  I'd like to see these at least load their defaults from the chrome metrics, but I unless you have a specific reason for wanting chrome.yaml-level overrides I think ti would be better to *only* load it from the metrics (no per-widget overrides).